rds-dc-with-one-vm-domainjoin/azuredeploy.json

```diff
@@ -0,0 +1,386 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+
+    "parameters": {
+
+        "location": {
+            "type": "string",
+            "defaultValue": "West US",
+            "allowedValues": [
+                "West US",
+                "East US",
+                "West Europe",
+                "East Asia",
+                "Southeast Asia"
+            ],
+            "metadata": {
+                "Description": "The region to deploy the resources into"
+            }
+        },
+
+        "publicDnsName": {
+            "type": "string",
+            "metadata": {
+                "Description": "Unique public DNS prefix for the deployment. The fqdn will look something like '<dnsname>.westus.cloudapp.azure.com'. Up to 62 chars, digits or dashes, lowercase, should start with a letter: must conform to '^[a-z][a-z0-9-]{1,61}[a-z0-9]$'."
+            }
+        },
+
+        "storageAccountName": {
+            "type": "string",
+            "metadata": {
+                "Description": "Name of the Storage Account. This has to be a uinique name, up to 24 chars, all lowercase."
+            }
+        },
+
+        "sourceImageName": {
```

Comment:
This method of getting image will soon be deprecated, this is a breaking change https://github.com/Azure/azure-quickstart-templates/blob/master/breaking-change.md Please move to the new API 2015-05-01